defaults/main.yml

```diff
@@ -48,6 +48,8 @@ haproxy_defaults_timeout:
     timeout: 50000
   - type: server
     timeout: 50000
+  - type: tunnel
+    timeout: "1h"
```

Comment:
Is this the default form the documentation?

Reply:
Not sure if it's the default as it's not specified in the documentation. But it's the one given in the example: http://cbonte.github.io/haproxy-dconv/2.2/configuration.html#4-timeout%20tunnel

Reply:
I'm not convinced that this should be added to `haproxy_defaults_timeout` (by default, unless it's now in the default haproxy config file). Off course you are free to override it yourself.